src/pypi2nix/stage2.py

```diff
@@ -22,7 +22,7 @@ def find_homepage(item):
     if 'extensions' in item and \
             'python.details' in item['extensions'] and \
             'project_urls' in item['extensions']['python.details']:
-        homepage = item['extensions']['python.details'].get('Home', '')
+        homepage = item['extensions']['python.details']['project_urls'].get('Home', '')
```

Comment:
@nlewo you should also check for existence of ``project_urls`` key, but other than that 👍